pyteal/ast/ecdsa.py

```diff
@@ -111,33 +114,39 @@ def EcdsaVerify(
     return EcdsaVerifyExpr(curve, data, sigA, sigB, pubkey[0], pubkey[1])
 
 
-def EcdsaDecompress(curve: EcdsaCurve, compressed_pk: Expr) -> MultiValue:
+class EcdsaDecompress(MultiValue):
```

Comment:
@algoidurovic Sanity checking:  In https://github.com/algorand/pyteal/pull/307, do you recall if there's a by-design why _only_ `EcdsaVerifyExpr` performs minimum version checks?

From my read of https://github.com/algorand/pyteal/pull/307, it seems we simply missed it.

Reply:
If I remember correctly, it was because the version checking needed to happen within the MultiValue class (ie all uses of MultiValue/MaybeValue had this issue). We decided it would be better to address that problem in a separate PR but it must've slipped my mind.

Reply:
@algoidurovic Thanks for the double check - resolving.